Add additional base test suite tests

loeakaodas opened the pull request:
# Description of change
added missing tests as per [TDL-14049](https://jira.talendforge.org/browse/TDL-14049)

# Manual QA steps
 - ran tap-tester locally
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

kspeer825 commented on tests/test_github_automatic_fields.py:
```diff
@@ -0,0 +1,80 @@
+        when no fields are selected and only the automatic fields are replicated.
+        """
+
+        expected_streams = set([
```
This should run off of the expected streams defined in base.py

loeakaodas replied:
bug added to Jira and indicated in the code.

kspeer825 commented on tests/test_github_automatic_fields.py:
```diff
@@ -0,0 +1,80 @@
+        """
+        Verify that for each stream you can get multiple pages of data
+        when no fields are selected and only the automatic fields are replicated.
+        """
```
This assertion is missing`Verify that all replicated records have unique primary key values`.  See here for test standards https://github.com/stitchdata/tap-tester/blob/master/reference/expectations/saas_taps.md#assertions-1

kspeer825 reviewed (changes requested):
Left a few comments

loeakaodas commented on tests/base.py:
```diff
@@ -83,7 +83,7 @@ def expected_check_streams():
             'stargazers',
             'team_members',
-            'team_memberships',
+            # 'team_memberships',
```
updated to use `expected_streams` method and exclude streams that are not testable.

kspeer825 reviewed (changes requested):
This is missing the proper .circleci/config.yml so that tests will run on schedule as regression protection. In addition the bug ticket linked in the test does not include any details of the failure or how to reproduce the issue.

loeakaodas commented:
@kspeer825 I wasn't sure what you meant by the circle config running on a schedule, but I adjusted the `config.yml` file to the latest image. I believe that it was already setup to run daily using a cron schedule.

Also added additional detail to the bug ticket.

kspeer825 reviewed (commented):
Is this PR still valid, or can it be closed?

loeakaodas commented:
Closing this PR, re-submitted in #145
